pkg-scripts/preinstall

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+cd ~/../../usr/local/bin/;
```

Comment:
Can this be an absolute path? Seems like it would be ... safer?
`/usr/local/bin/`